cvat-ui/src/components/header/settings-modal/player-settings.tsx

```diff
@@ -67,13 +75,23 @@ export default function PlayerSettingsComponent(props: Props): JSX.Element {
         onChangeBrightnessLevel,
         onChangeContrastLevel,
         onChangeSaturationLevel,
+        onChangeCanvasBackgroundColor,
     } = props;
 
     const minFrameStep = 2;
     const maxFrameStep = 1000;
     const minGridSize = 5;
     const maxGridSize = 1000;
 
+    const imageRef = useRef<HTMLImageElement>(null);
+
+    useEffect(() => {
+        if (imageRef.current) {
+            imageRef.current.style.filter = `brightness(${brightnessLevel / 100})`
+                + `contrast(${contrastLevel / 100})`
+                + `saturate(${saturationLevel / 100})`;
+        }
+    }, [imageRef, brightnessLevel, contrastLevel, saturationLevel]);
```

Comment:
Only props must be here: https://ru.reactjs.org/docs/hooks-reference.html#useeffect
imageRef is a local variable

Reply:
Strange behavior here. If I open an annotation page without visiting dashboard, preview is always empty.
After I've visited dashboard I always have the same preview image even when go to another task.

Reply:
@bsekachev, yes, i simply pick first preview from task. And if there is no tasks list, in case user visit only task annotation page, we have no such list so we won't render preview.

Reply:
@ActiveChooN 
I am not sure this is a nice way. Looks really confusing

Reply:
@bsekachev, if now settings present in separate modal, maybe we can simply remove it?

Reply:
@ActiveChooN 
I think yes. It isn't essential for now